src/uhttp.c

```diff
@@ -834,7 +834,8 @@ static int construct_http_headers(HTTP_HEADERS_HANDLE http_header, size_t conten
         }
         if (!hostname_found)
         {
-            size_t host_len = strlen(HTTP_HOST)+strlen(hostname)+8+2;
+            // calculate the size of the host header
+            size_t host_len = strlen(HTTP_HOST)+strlen(hostname)+MAX_CONTENT_LENGTH+2;
```

Comment:
>strlen(HTTP_HOST)+strlen(hostname)+MAX_CONTENT_LENGTH+2 [](start = 30, length = 55)

nit: can we put some spaces between the elements and the `+`'s? so
`strlen(...) + MAX_CONTENT_LENGTH + ...`